spec/support/factory_girl.rb

```diff
@@ -28,4 +29,8 @@
     user
   end
 
+  factory :category do
+    topic 'Ruby'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.